### PR TITLE
Add basic entry card in the qc page

### PR DESCRIPTION
**Pull request by frozenhelium**
Depends on: https://github.com/the-deep/server/pull/517

### CODE REVIEW

File: src/components/viewer/DateOutput/index.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import { _cs } from '@togglecorp/fujs';
+
+import FormattedDate from '#rscv/FormattedDate';
+import Icon from '#rscg/Icon';
+
+import styles from './styles.scss';
+
+
+export interface DateOutputProps {
+    className?: string;
```
Comment (AdityaKhatri) [inline src/components/viewer/DateOutput/index.tsx:11]: Support title on hover
Also, add title show/hide behavior

File: src/views/Entries/requests/EntriesRequest.js [hunk 2]
```diff
@@ -23,6 +23,8 @@ export default class EntriesRequest extends Request {
             count: totalEntriesCount,
         } = response;
 
+        console.info(responseEntries);
```
Comment (tnagorra) [inline src/views/Entries/requests/EntriesRequest.js]: Let's not leave console.info intended for debugging.

File: src/constants/routes.js [hunk 3]
```diff
@@ -167,7 +167,7 @@ export const routes = {
     editEntries: {
         order: 41,
         type: ROUTE.private,
-        path: '/projects/:projectId/leads/:leadId/entries/edit/',
+        path: '/projects/:projectId/leads/:leadId/entries/:entryId?/edit/',
```
Comment (AdityaKhatri) [inline src/constants/routes.js]: This is not necessary right?

File: src/views/Entries/QualityControl/index.tsx [hunk 4]
```diff
@@ -29,7 +37,22 @@ function QualityControl(props: QualityControlProps) {
                 rightContainerClassName={styles.right}
                 rightChild={(
                     <div className={styles.entryList}>
-                        Entry list
+                        { leadGroupedEntriesList.map((leadWithEntries) => {
```
Comment (AdityaKhatri) [inline src/views/Entries/QualityControl/index.tsx:40]: Use list components

File: src/views/Entries/requests/EntriesRequest.js (hunk 2 above)
Comment (AdityaKhatri) [inline src/views/Entries/requests/EntriesRequest.js]: remove this

File: src/typings/entry.ts [hunk 5]
```diff
@@ -0,0 +1,62 @@
+import { DatabaseEntityBase } from './common';
+
+type EntryType = 'excerpt' | 'image';
```
Comment (tnagorra) [inline src/typings/entry.ts]: we should also consider another type 'dataSeries'

File: src/constants/routes.js (hunk 3 above)
Comment (tnagorra) [inline src/constants/routes.js]: I think we should revert this if we are not using this!

File: src/components/viewer/DateOutput/index.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    value: string;
```
Comment (tnagorra) [inline src/components/viewer/DateOutput/index.tsx]: We should handle case where value is `undefined`

File: src/views/Entries/QualityControl/index.tsx (hunk 4 above)
Comment (frozenhelium) [reply src/views/Entries/QualityControl/index.tsx:40]: This is only temporary and will be changed later

File: src/components/viewer/DateOutput/index.tsx (hunk 1 above)
Comment (frozenhelium) [reply src/components/viewer/DateOutput/index.tsx:11]: done

**Pull request merged**